kitsune/users/jinja2/users/edit_profile.html

```diff
@@ -27,24 +27,28 @@
     <h1 class="sumo-page-heading">{{ _('Your Account') }}</h1>
     <p class="mb-0"><strong>{{ user.email }}</strong></p>
     <p>
-      {% trans a_open='<a href="https://support.mozilla.org/en-US/kb/change-primary-email-address-firefox-accounts">'|safe, a_close='</a>'|safe %}
-        To change your email or avatar, visit the Mozilla accounts page. {{ a_open }} Learn more. {{ a_close }}
+      {# L10n: Learn more refers to https://support.mozilla.org/kb/change-primary-email-address-firefox-accounts #}
+      {% trans a_open='<a href="https://support.mozilla.org/kb/change-primary-email-address-firefox-accounts">'|safe, a_close='</a>'|safe %}
+        To change your email or avatar, visit the Mozilla accounts page. {{ a_open }}Learn more.{{ a_close }}
```

Comment:
I'm afraid the whitespace change would export new strings, removing all existing translations; which was sorta my r- on the previous PR.